system/Database/OCI8/Builder.php

```diff
@@ -222,7 +222,7 @@ protected function _limit(string $sql, bool $offsetIgnore = false): string
         }
 
         $this->limitUsed    = true;
-        $limitTemplateQuery = 'SELECT * FROM (SELECT INNER_QUERY.*, ROWNUM RNUM FROM (%s) INNER_QUERY WHERE ROWNUM < %d)' . ($offset ? ' WHERE RNUM >= %d' : '');
+        $limitTemplateQuery = 'SELECT * FROM (SELECT INNER_QUERY.*, ROWNUM RNUM FROM (%s) INNER_QUERY WHERE ROWNUM < %d)' . ($offset !== 0 ? ' WHERE RNUM >= %d' : '');
```

Comment:
negative is truthy so `!==0` is used, ref https://3v4l.org/dajZg

Reply:
`$offset` is non negative.

Reply:
I check on another part on `BaseBuilder`, `$offset !== 0` is used so I think `!==0` is valid.

Reply:
`$offset` is non negative. So `!== 0` is no problem.
But it is the same as `> 0`.

Reply:
let use `!==0` for consistency